Dockerfile

```
@@ -43,7 +43,7 @@ RUN git clone https://github.com/njoy/NJOY2016 /opt/NJOY2016 && \
 
 # Clone and install OpenMC without DAGMC
 RUN if [ "$include_dagmc" = "false" ] ; \
-    then git clone --recurse-submodules https://github.com/openmc-dev/openmc.git ; \
+    then git clone --recurse-submodules https://github.com/openmc-dev/openmc.git \
     /opt/openmc ; \
```

I think what you have is fine but as an option perhaps this makes it clearer

```suggestion
    then git clone --recurse-submodules https://github.com/openmc-dev/openmc.git /opt/openmc ; \
```

---

Good point. Happy to oblige!